src/com/algosolution/AlgoSolution.java

```diff
@@ -0,0 +1,23 @@
+package com.algosolution;
+
+
+
+public class AlgoSolution {
+    public static void FIND(int Start, int End, String Text) {
+        String[] MonTableau = Text.split("");
```

Comment:
Same here. You can use 
Lowercase: montableau
CamelCase: monTableau
Snake_case: mon_tableau